src/data/messaging.js

```diff
@@ -0,0 +1,43 @@
+import { writable } from 'svelte/store'
+import { getMessageCounts, markMessagesAsRead } from './gqlQueries'
+
+export const unreads = writable([])
+
+const EVERY_MINUTE = 1000
+setInterval(loadMessageCounts, EVERY_MINUTE)
+
+async function loadMessageCounts() {
+  const unread = ({ unreadMessageCount }) => unreadMessageCount > 0
```

Comment:
Maybe consider including a verb in these function names:
```suggestion
  const excludeRead = ({ unreadMessageCount }) => unreadMessageCount > 0
```